src/Front/Pattern/Desugar.hs

```diff
@@ -90,6 +90,7 @@ desugarExprs = mapM desugarExpr
 desugarExpr :: A.Expr -> DesugarFn A.Expr
 desugarExpr e@(A.Id  _       ) = return e
 desugarExpr e@(A.Lit _       ) = return e
+desugarExpr e@(A.ListExpr _       ) = return e
```

Comment:
pretty sure this should map desugarExpr over elements in the list. I will fix this real quick along with the newlines.